build.sbt

```diff
@@ -1,12 +1,12 @@
 import com.typesafe.sbt.packager.docker.Cmd
 
-val scala212 = "2.12.15"
-val scala213 = "2.13.7"
+val scala212 = "2.12.14"
+val scala213 = "2.13.6"
```

Comment:
Please add some comments about these version requirements!

Reply:
I added it directly in your PR https://github.com/cognitedata/cdp-spark-datasource/pull/538/files#r748148476 ;)